src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+
+  &__star-active {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    box-sizing: border-box;
+    align-items: center;
```

Comment:
The 'align-items' property is not necessary here since '.stars__star-active' is not a flex container. The stars are individual items within a flex container, and their alignment should be controlled by the parent container '.stars'.